tests/queries/0_stateless/01052_window_view_proc_tumble_to_now.sh

```diff
@@ -8,25 +8,25 @@ opts=(
     "--allow_experimental_analyzer=0"
 )
 
+db="$(random_str 10)"
+
 $CLICKHOUSE_CLIENT "${opts[@]}" --multiquery <<EOF
 SET allow_experimental_window_view = 1;
-DROP TABLE IF EXISTS mt;
-DROP TABLE IF EXISTS dst;
-DROP TABLE IF EXISTS wv;
+DROP DATABASE IF EXISTS ${db};
```

Comment:
This should be automatic. Each test runs under `$CLICKHOUSE_DATABASE`. If we need to manually create it it's either because we are doing extremely special (which we aren't in this case), a bad test or a bug.